src/tests/simulation/src/cluster.rs

```diff
@@ -93,6 +94,9 @@ pub struct Configuration {
 
     /// Queries to run per session.
     pub per_session_queries: Arc<Vec<String>>,
+
+    /// dir to store SQL backend sqlite db
+    pub sqlite_data_dir: Option<PathBuf>,
```

Comment:
Can we use a temporary file (or even in-memory) for SQLite and make this a boolean flag instead?

Reply:
Agree. Because we run multiple simulation tests parallelly, you might need to set this argument carefully.

Reply:
There's a uuid suffix currently. Improve in future PR. I want to use sqlite in-memory store.